frontend/src/AddMovie/AddNewMovie/AddNewMovieSearchResult.js

```diff
@@ -103,7 +103,7 @@ class AddNewMovieSearchResult extends Component {
                   <MoviePoster
                     className={styles.poster}
                     images={images}
```

Comment:
pass in `style=` prop here with height and width per discord discussion for consistency.